src/mbgl/util/offscreen_texture.cpp

```diff
@@ -36,15 +39,31 @@ class OffscreenTexture::Impl {
         return size;
     }
 
+    void bindRenderbuffers() {
+        if (!framebuffer) {
+            texture = context.createTexture(size, gl::TextureFormat::RGBA, unit);
+            colorTarget = context.createRenderbuffer<gl::RenderbufferType::RGBA4>(size);
+            depthTarget = context.createRenderbuffer<gl::RenderbufferType::DepthComponent>(size);
+            framebuffer = context.createFramebuffer(*colorTarget, *depthTarget, *texture);
+        } else {
+            context.bindFramebuffer = framebuffer->framebuffer;
+        }
+
+        context.viewport = { 0, 0, size };
+    }
+
 private:
     gl::Context& context;
     const Size size;
+    gl::TextureUnit unit;
     optional<gl::Framebuffer> framebuffer;
     optional<gl::Texture> texture;
+    optional<gl::Renderbuffer<gl::RenderbufferType::RGBA4>> colorTarget;
```

Comment:
Why do we add an additional color target? The point of this class is to draw to the texture.